File: src/main/java/com/patneh/shelter/infrasructure/enums/MiejsceEnum.java

package com.patneh.shelter.infrasructure.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
public enum MiejsceEnum {
    LECZNICA("Lecznica"),
    SCHRONISKO("Schronisko"),
    DOM_TYMCZASOWY("Dom_tymczasowy"),
    DOM_STALY("Dom_staly");

    @Getter
    private final String label;
}

File: shelter-frontend/src/app/shared/enums/miejsce.enum.ts
export enum MiejsceEnum {
  LECZNICA = 'LECZNICA',
  SCHRONISKO = 'SCHRONISKO',
  DOM_TYMCZASOWY = 'DOM_TYMCZASOWY',
  DOM_STALY = 'DOM_STALY',
}

export const MiejsceLabels: Record<MiejsceEnum, string> = {
  [MiejsceEnum.LECZNICA]: 'Lecznica',
  [MiejsceEnum.SCHRONISKO]: 'Schronisko',
  [MiejsceEnum.DOM_TYMCZASOWY]: 'Dom tymczasowy',
  [MiejsceEnum.DOM_STALY]: 'Dom staly'
};



